app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -4,6 +4,13 @@ class Api::V2::HostSubscriptionsController < Katello::Api::V2::ApiController
     before_filter :check_subscriptions, :only => [:add_subscriptions, :remove_subscriptions]
     before_filter :find_content_view_environment, :only => :create
 
+    def_param_group :subscription_facet_attributes do
+      param :release_version, String, :desc => N_("Release version for this Host to use (7Server, 7.1, etc)")
+      param :autoheal, :bool, :desc => N_("Sets whether the Host will autoheal subscriptions upon checkin")
+      param :service_level, Integer, :desc => N_("Service level to be used for autoheal.")
+      param :hypervisor_guest_uuids, Array, :desc => N_("List of hypervisor guest uuids")
```

Comment:
param :installed_products, Array, :desc => N_("List of products installed on the host") do
      param :product_id, String, :desc => N_("Product id as listed from a host's installed products, \
        this is not the same product id as the products api returns")
      param :product_name, String, :desc => N_("Product name as listed from a host's installed products")
      param :arch, String, :desc => N_("Product architecture")
      param :version, String, :desc => N_("Product version")
    end